### ISSUE TITLE
[Feature Request] Video thumbnails/ability to stream all MP4's from the web

### ISSUE THREAD

**Issue by ialexpw** [FilenCloudDienste/filen-drive#16]
I was told this may have been a feature in the past, but it no longer works for video files.

It would be very handy to have thumbnails for video files (MP4's etc) as if you have a lot of videos within a folder with not-so-easy names, it can be quite painful to work which is which.

As a side note to this, only some of the MP4 files can be streamed from the web interface, so it can be even more difficult to sort/rename them.

**Comment by ialexpw** [comment]
Yes just noticed this and can see it added in the latest changes.

Only other thing to mention from the title is about streaming bigger MP4 files via the web interface - seems to be limited ~150MB at the moment.

**Comment by Dwynr** [comment]
Video thumbnails have been added, working on video streaming. Will take a longer.

**Comment by ialexpw** [comment]
> Video thumbnails have been added, working on video streaming. Will take a longer.

Thank you, it's much better! - as above I am still hitting issues with some video thumbnails - most generate fine, but some will generate the errors above in the console and never create them.

And great to hear about the streaming. 😄
